app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy(command: str) -> None:
+    cmd, file_name, new_file_name = command.split()
+    if file_name == new_file_name or cmd != "cp":
+        print("Does nothing")
```

Comment:
Your function don't stop here. You can take in mind my higher comment and do that way or at least make this version work :upside_down_face: